extensions/datasource/deployment/src/main/java/io/quarkus/datasource/deployment/DataSourceProcessor.java

```diff
@@ -0,0 +1,5 @@
+package io.quarkus.datasource.deployment;
+
+public class DataSourceProcessor {
```

Comment:
Is this leftover or something?

Reply:
No. IIRC, the javadoc plugin complains when releasing when an artifact is entirely empty.

Reply:
Can we then call it something other than `DataSourceProcessor`? It just seems a little missleading to someone who reads it for the first time. 
Otherwise a tiny bit of Javadoc would be nice